l10n_it_fatturapa_in_notifications/account.py

```diff
@@ -19,11 +19,37 @@
 ##############################################################################
 
 from openerp.osv import fields, orm
+import netsvc
 
 
 class AccountInvoice(orm.Model):
     _inherit = "account.invoice"
     _columns = {
         'result_notification_id': fields.many2one(
-            'fatturapa.notification', "Result notification", readonly=True)
+            'fatturapa.notification', "Result notification", readonly=True),
+        'rejected': fields.boolean('Rejected')
         }
+
+    _defaults = {
+        'rejected': False
+    }
+
+    def action_cancel_draft(self, cr, uid, ids, *args):
+        if not hasattr(ids, '__iter__'):
+            ids = [ids]
+        invoice = self.browse(cr, uid, ids[0])
```

Comment:
non mi sembra che il reject possa essere fatto in massa dato che necessita di un messaggio di commento. E poi per come e' fatto action_cancel_draft originale non vedo come sia possibile, in ogni caso non mi sembra sia possibile fare tramite la vista un cancel delle fatture di massa.
ti risulta?